src/binaryen-c.cpp

```diff
@@ -1102,7 +1136,8 @@ int BinaryenModuleValidate(BinaryenModuleRef module) {
   }
 
   Module* wasm = (Module*)module;
-  return WasmValidator().validate(*wasm) ? 1 : 0;
+  FeatureSet features = Feature::Atomics;
```

Comment:
this is fine for now I guess, but please add a comment `// TODO add feature selection support to C API`, as we do need to add that eventually.